docs/metrics.md

```diff
@@ -152,6 +152,12 @@ An event fired when the migration from Quill.JS to CKEditor already happened
 - `ec` - `notes`
 - `ea` - `metrics-migrated-before`
 
+#### `metrics-context-menu`
```

Comment:
this is gonna be just `context-menu` i think, per `background.js` file

Reply:
so in Google analytics it will display the event as `'context-menu'` because panel js sends a ``metrics-context-menu`` and that calls `sendMetrics` `'context-menu'`.

(which later calls `return analytics.sendEvent('notes', event, metrics);` where `event=context-menu`